shared/profile/edit-profile/index.js

```diff
@@ -0,0 +1,79 @@
+/* @flow */
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import {editProfile} from '../../actions/profile'
+import {maxProfileBioChars} from '../../constants/profile'
+
+import Render from './render'
+import type {Props} from './render'
+import {navigateUp} from '../../actions/router'
+
+type State = {
+  bio: string,
+  fullname: string,
+  location: string
+}
+
+class EditProfile extends Component<void, Props, State> {
+  props: Props;
+  state: State;
+
+  static parseRoute (currentPath, uri) {
```

Comment:
Not sure why I need an empty parseRoute here, when the component's being passed as a subroute of its parent.. any ideas?